.Lib9c.Plugin/Lib9c.Plugin.csproj

```diff
@@ -15,5 +15,48 @@
       <ExcludeAssets>runtime</ExcludeAssets>
     </ProjectReference>
   </ItemGroup>
-  
+
+  <ItemGroup>
+    <ProjectReference Include="..\.Libplanet\src\Libplanet.Action\Libplanet.Action.csproj">
+      <Private>false</Private>
+      <ExcludeAssets>runtime</ExcludeAssets>
+    </ProjectReference>
+
+    <ProjectReference Include="..\.Libplanet\src\Libplanet\Libplanet.csproj">
+      <Private>false</Private>
+      <ExcludeAssets>runtime</ExcludeAssets>
+    </ProjectReference>
+    <ProjectReference Include="..\.Libplanet\src\Libplanet.Types\Libplanet.Types.csproj">
+      <Private>false</Private>
+      <ExcludeAssets>runtime</ExcludeAssets>
+    </ProjectReference>
+    <ProjectReference Include="..\.Libplanet\src\Libplanet.Common\Libplanet.Common.csproj">
+      <Private>false</Private>
+      <ExcludeAssets>runtime</ExcludeAssets>
+    </ProjectReference>
```

Comment:
These assemblies need to be included because it will try to use shared types if these assemblies are excluded and it will cause a type mismatch error. If not, in my opinion, Libplanet must guarantee type backward-compatibility by SemVer strategy, and Lib9c have to be able to ignore new Libplanet major updates. (no bump)